PyHa/statistics.py

```diff
@@ -263,25 +263,25 @@ def automated_labeling_statistics(
                 if statistics_df.empty:
                     statistics_df = clip_stats_df
                 else:
-                    statistics_df = statistics_df.append(clip_stats_df)
+                    statistics_df = pd.concat([statistics_df,clip_stats_df])
             elif stats_type == "IoU":
                 IoU_Matrix = clip_IoU(clip_automated_df, clip_manual_df)
                 clip_stats_df = matrix_IoU_Scores(
                     IoU_Matrix, clip_manual_df, threshold)
                 if statistics_df.empty:
                     statistics_df = clip_stats_df
                 else:
-                    statistics_df = statistics_df.append(clip_stats_df)
+                    statistics_df = pd.concat([statistics_df, clip_stats_df])
         except BaseException as e:
```

Comment:
E4E pylint discourages broad-except.  Please consider not catching `builtins.BaseException`
E4E pylint does not allow `e` as a variable name.  Please consider `exc` or `exp`